promql/fuzz.go

```diff
@@ -68,6 +68,10 @@ func fuzzParseMetricWithContentType(in []byte, contentType string) int {
 		panic(warning)
 	}
 
+	if contentType == "application/openmetrics-text" {
+		p = textparse.NewOpenMetricsParser(in, symbolTable)
+	}
```

Comment:
Why?  This appears to be what the code above already does.

Reply:
this will be removed in my PR [here](https://github.com/prometheus/prometheus/pull/14738/files#diff-101f8eadd484ec6582b1c1973be770a7e35bd388abc21e809898c055ef3c38f3R64)

But this looks like its a relic from an earlier implementation in this PR that I forgot to remove: https://github.com/prometheus/prometheus/pull/14356#discussion_r1682815624